src/params.rs

```diff
@@ -254,9 +254,6 @@ where
     /// ```no_run
     /// # use stripe::{Customer, ListCustomers, StripeError, Client};
     /// # use futures_util::TryStreamExt;
-    /// # fn main() {
-    /// # tokio_test::block_on(run());
-    /// # }
```

Comment:
This is marked `no_run` so there should be no user-visible change by removing this.